README.md

```diff
@@ -6,67 +6,109 @@ This repository hosts the new IATI website based on Django and Wagtail CMS.  A P
 The current scope of the project (to April 2018) focuses on the 'About IATI' and 'Publisher guidance' sections.
 
 
-## Pre-requites
+## Local Development
 
-- Python3
-- PostgreSQL
+### Pre-requites
 
+- Docker _(See Docker documentation for installation instructions per OS [https://docs.docker.com/install/](https://docs.docker.com/install/))_
+- Docker Compose
 
-## Dev setup
+**Important:** Ensure that native Postgres and Elasticsearch services are stopped. Docker will attempt to use these ports for its own services.
 
-Set-up and activate virtual environment
-```
-python3 -m venv pyenv
-source pyenv/bin/activate
-```
+
+### Dev setup
 
 Enter into the Django project directory
 ```
 cd iati
 ```
 
-Install requirements
+Copy the example local settings file and enter database settings accordingly.
```

Comment:
is this part still relevant? a little bit below `dev.py` is mentioned as the main settings file

I just removed `local.py` and re-run the entire setup without it, all works fine, so maybe this part is now obsolete?

Reply:
That's completely right! It can be removed